understanding/20/name-role-value.html

```diff
@@ -70,12 +70,6 @@ <h2>Examples of Name, Role, Value</h2>
    <section id="resources">
       <h2>Resources for Name, Role, Value</h2>
       <ul>
-         <li>
-            <a href="https://www.w3.org/WAI/PF/roadmap/">Dynamic Accessible Web Content Roadmap</a>
-         </li>
-         <li>
-            <a href="https://www.w3.org/WAI/PF/GUI/">Role Taxonomy for Accessible Adaptable Applications</a>
-         </li>
          <li>
             <a href="https://www.w3.org/TR/wai-aria/">Web Accessibility Initiative - Accessible Rich Internet Applications (ARIA)</a>
          </li>
```

Comment:
```suggestion
         </li>
         <li>
             <a href="https://www.w3.org/TR/html-aria/">ARIA in HTML</a>
         </li>
         <li>
             <a href="https://www.w3.org/TR/accname-1.2/">Accessible Name and Description Computation 1.2</a>
         </li>
```

@momdo are you happy to add these?

Reply:
@patrickhlauke 
Looks good to me

Reply:
great stuff, thanks :)